Add seo blog (#47)
apiobuild/apiobuild.github.io: content/blog/website-is-not-the-seo-you-think.md

---
title: "Website-Building is not the SEO You Think"
image: /images/blog/web_search.png
date: 2020-08-09
tags: ["chopin", "blog","seo", "marketing", "google", "website"]
---

> SEO is one of those trendy keywords that every website developers put on their ads. Many people came to us and asked if our website can do SEO. SEO seems to be the magic word that will bring you thousands of sales and people think the first step is to get a website. Why does everyone think building a website would give them SEO? **In this article, let us breakdown this fancy term and tell you [how apio creates SEO-friendly websites](#chopin-s-seo-features).** And no, building a website is not the SEO you think.

Reviewer: ```suggestion
SEO is one of those trendy words that every website developer lists on their offerings. Many people come to us and ask if our website can do SEO. **In this article, let us breakdown this fancy word and discuss [how apio creates SEO-friendly websites](#chopin-s-seo-features).** Our honest answer? No, building a website does not give you SEO.
```

🖥 [Read more: What you need from developers when it comes to website-building?](https://apiobuild.com/blog/build-a-website-for-your-business-what-do-you-need/)

🖥 [Learn more: apio\'s static online store generator - Chopin](https://apiobuild.com/blog/create-an-online-store-for-free/#introducing-chopin)

## TL; DR

> **SEO is driven by content! Website developers give you a framework to publish content not the content itself.** 

> Owner has to create great content to optimize SEO. [Wordpress](https://wordpress.org/) (or any other website builders) does not give you "SEO", the content you publish on wordpress gives you SEO. Why would you ask a web developer/design agency to do this - go to marketing consultancy.

Reviewer: ```suggestion
Owner has to create great content to optimize SEO. [Wordpress](https://wordpress.org/) (or any website builder) does not give you "SEO", the content you publish gives you SEO. Why would you ask a web developer/design agency to do this - go to marketing agency.
```

## What is SEO?

> SEO (Search Engine Optimization) is using organic traffic generation by the website’s content (blog articles) to make business or brand to be visible for searchers on search engines (like [Google](https://www.google.com/), [Yahoo](https://www.yahoo.com/), [Bing](https://www.bing.com/)). Google did not reveal how exactly their algorithms determine where to position your website in the search results, but there are some factors that will improve your search result ranking:

Reviewer: ```suggestion
SEO (Search Engine Optimization) makes business or brand to be visible on search engines (like [Google](https://www.google.com/), [Yahoo](https://www.yahoo.com/), [Bing](https://www.bing.com/)) by driving organic traffic with contents (blog, articles). Google did not reveal how exactly their algorithms determine where to position your website in the search results, but there are many factors that will improve your website's ranking:
```

>  - [Relevancy](#relevancy): **How much is your content related to the keyword?** When people search "custom cake delivery in New York", they will not find a realtor's website.

Reviewer: ```suggestion
 - [Relevancy](#relevancy): **How relevant is your content to the keyword?** When people search "custom cake delivery in New York", they will not find a realtor's website.
```

>  - [Authority](#authority): **How many people find your content credible?** When more people link your page to their sites, Google think your website is authoritative. 

Reviewer: ```suggestion
 - [Authority](#authority): **How many people find your content credible?** When more people link your page to their sites, Google thinks your website is authoritative. 
```

>  - [Usefulness](#usefulness): **Is your content easy to understand so people can actually utilize that information?** You rarely find academic thesis in your search results, even when you search some technical terms. If you can't understand the content, then that's not useful for you.

Reviewer: ```suggestion
```

## Developer vs. Marketer 

> So, how can developers help you to achieve SEO? Where do marketing specialists come in?

Reviewer: ```suggestion
How can developers help you to achieve SEO? Where do marketing specialists come in?
```

> Developers should build you a **SEO-friendly website framework** that helps search engine spiders find and index all of the pages on your website. Marketers should help you identify keywords and polish the content that can drive traffic to your website.

Reviewer: ```suggestion
Developers should build you an **SEO-friendly website framework** that helps search engine spiders find and index all the pages on your website. Marketers should help you identify keywords and polish the content that can drive traffic to your website.
```

#### Relevancy

> 💻 Developer builds a website framework that **indexes your site** properly so search engines can read the content and eventually determine if that's relevant to the search.

Reviewer: ```suggestion
💻 Developer builds a website framework that **indexes your site** allowing search engines to read the content and calculate relevancy to the search.
```

📊 Marketer conducts keyword reseach and helps you **identify popular or niche keyphrases**.

#### Authority 

💻 Developer makes sure your website content is **linkable**.

📊 Marketerg **generates trustworthy contents** that other sites would link to. For example, if you write a detailed article to explain the difference between baking soda and baking powder, it's likely that baking blogger would link your page as a reference.

> #### Usefulness

Reviewer: As mentioned, no such metric in page rank.

💻 Developer sets up a **simple and flat website architecture** so users can find information easily and in less than 4 clicks. 

📊 Marketer **develops relatable contents** that people can actually understand. Sometimes copywriters help you to revise technical jargons and rephrase in your target audience's language.

<img src="/images/blog/dev-vs-marketer.png" class="post-img">

## Chopin's SEO Features

> [Chopin is apio\'s static online store generator](https://telescope.apiobuild.com/app/chopin). Despite how easy it is to set up an online store, we offer a SEO-optimized website framework and take care many of the SEO integration already.

Reviewer: ```suggestion
[Chopin is apio\'s static online store generator](https://telescope.apiobuild.com/app/chopin). Despite how easy it is to set up an online store, we offer an SEO-optimized website framework and gives you many SEO integrations.
```

- **Meta titles & descriptions**: We build-in meta title and summary of your site that gets shown in Google search results. [Your store name and description](https://apiobuild.com/blog/how-to-create-web-store-with-apio/#step-4-create-the-store) will translate to this meta data.

> - **Index**: We index the online store pages with users' consent, so your store's story and products can be read by search engines.

Reviewer: ```suggestion
- **Index**: We index the online store pages upon user consent. Your store's story and products can be found easily by search engines.
```

- **Social Media Links**: With built-in meta descriptions, we ensure the page thumbnail or link preview looks professional when your share your store on social medias.

> - **Fast Site Speed**: Website users can be impatient. Chopin-generated sites are static and lean, which means it takes less time to load comparing to normal sites.

Reviewer: ```suggestion
- **Fast Site Speed**: Website users can be impatient. Chopin-generated sites are static and lean, we constantly load test loading time to ensure performant response time.
```

Reviewer: what's normal sites?

> - **Mobile-Friendliness**: [A responsive website](https://apiobuild.com/blog/create-an-online-store-for-free/#responsive-design) means beyond better user experience in 2020. Mobile is how your prospective customers are searching for and browsing your site, and mobile-friendliness is a confirmed ranking factor.

Reviewer: ```suggestion
- **Mobile-Friendly**: [A responsive website](https://apiobuild.com/blog/create-an-online-store-for-free/#responsive-design) means better user experience. Mobile drives the the majority of the traffic. Mobile-friendliness is another important factor that improves ranking.
```

> - **Google Analytics**: If you want to dive more into the SEO game and analyze your website traffic data, we can help you integerate your by simply providing the tracking id.

Reviewer: ```suggestion
- **Google Analytics**: You can easily add google analytics-id and start analyze your page traffic.
```

Here's how our [demo dessert store](https://tinyurl.com/apio-demo-store) looks like on Google search results and on Facebook:

<img src="/images/blog/link-preview.png" class="post-img">

🛍️[Read more: follow this tutorial and create your online store today!](https://apiobuild.com/blog/how-to-create-web-store-with-apio/)

> ## But, do you really need SEO?

Reviewer: ```suggestion
## Do you really need SEO?
```

> Beyond those built-in features powered by Chopin, what else you need to do for SEO? SEO is for driving steady traffic to your website. If you are a lifestyle blogger making referrals to amazon products, yes, you want SEO. If you are a small restaurant or local business owner, there's no value (nor reason) having someone come to read your website every other day. 

Reviewer: ```suggestion
Beyond these built-in features powered by Chopin, what else you need to do for SEO? SEO is for driving steady traffic to your website. If you are a lifestyle blogger making referrals to amazon products, yes, you want SEO. If you are a small restaurant or local business owner, there's no value (nor reason) having someone come to read your website every other day. 
```
